images/macos/scripts/tests/Common.Tests.ps1

```diff
@@ -146,3 +146,23 @@ Describe "Colima" -Skip:($os.IsVentura -or $os.IsSonoma -or $os.IsSequoia) {
         "colima version" | Should -ReturnZeroExitCode
     }
 }
+
+Describe "Apache Ant" {
+    It "Apache Ant" {
+        "ant -version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Kotlin" {
+    $kotlinPackages = @("kapt", "kotlin", "kotlinc", "kotlinc-jvm", "kotlin-dce-js")
+
+    It "<toolName> is available" -TestCases ($kotlinPackages | ForEach-Object { @{ toolName = $_ } }) {
+        "$toolName -version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "sbt" {

```

Comment:
Here we need something like ` -Skip:(-not $os.IsMonterey)` to prevent additional runs for all three cases.